# Generated by Django 3.1.4 on 2021-03-18 10:05

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0075_auto_20210316_1223'),
    ]

    operations = [
        migrations.CreateModel(
            name='QualityIssue',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('deleted', models.DateTimeField(editable=False, null=True)),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('updated_at', models.DateTimeField(auto_now=True)),
                ('status', models.CharField(choices=[('under_review', 'Under Review'), ('removed', 'Removed'), ('contested', 'Contested'), ('in_progress', 'In Progress'), ('requesting_approval', 'Requesting Approval'), ('closed_out', 'Closed Out')], default='under_review', max_length=255)),
                ('description', models.TextField()),
                ('attachments', models.ManyToManyField(to='api.Media')),
                ('location_matrix', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='api.locationmatrix')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'db_table': 'quality_issue',
            },
        ),
    ]
